### PR TITLE
N20: add egregoria updates

### PR DESCRIPTION
**Pull request by Uriopass**
Related to #531

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/posts/newsletter-020/index.md [hunk 1]
```diff
@@ -64,6 +64,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+is a simulation oriented city builder that tries
+to replicate modern society as well as possible.
+
+The [8th devlog][egregoria-blog-post] was published.
+Updates include:
+
+ - Multiplayer based on deterministic lockstep
+ - Economy revamp inspired by Anno 1800
+ - Infinite world using procedural generation
+ - Many more QoL features
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:82]:
```suggestion
- Multiplayer based on deterministic lockstep
- Economy revamp inspired by Anno 1800
- Infinite world using procedural generation
- Many more QoL features
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
One minor formatting issue flagged up by the linter:

File: content/posts/newsletter-020/index.md (hunk 1 above)
Comment (Uriopass) [reply content/posts/newsletter-020/index.md:82]: Ah, my bad! I wanted to install a markdown linter but Intellij has no such plugin.

Comment (Uriopass) [reply content/posts/newsletter-020/index.md]: fixed. :-)

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by ozkriff** [comment]
:construction:

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**